### DISCUSSION TITLE
[Help Wanted] Wiki Page on WSJT-X Intergration

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#788] [General]
Hi All,

I was wondering if anyone who uses WSJT-X and Cloudlog or perhaps a few people were interested in working together to create some content on how to do it using Cloudlog. I've created a wiki page at https://github.com/magicbug/Cloudlog/wiki/WSJT-X-Integration 

I'm fairly clueless about these tools as its not really a mode I enjoy so I feel its probably better coming from someone who loves WSJT-X!

Peter, 2M0SQL

**Comment by timkirby** [comment]
Oh! I can probably help with this. I use ADIFPUSH so could cover that approach.

**Reply by magicbug** [reply]
> Oh! I can probably help with this. I use ADIFPUSH so could cover that approach.

Fab 👌 please!

**Comment by ghost** [comment]
Gridtracker supposedly supports Cloudlog, the flow being WSJT-X —> Gridtracker —> Cloudlog. The API test function in Gridtracker passed yet nothing got logged automatically when I tried. I’ll try to get that working and I’ll add my notes to the page.

**Reply by magicbug** [reply]
Thanks, while for some it's simple I get asked a lot how to do it, so a page on wsjtx doesn't seem terrible

**Reply by ghost** [reply]
All done ✅

**Comment by KeyOz** [comment]
I'm using some self written Python script on my vserver. WSTJ-X sends the log data to this server. This serverscript send a MQTT-Message to an MQTT-Broker on that server and an other script received the log-data from that MQTT-Broker and sends it to my Cloudlog. I'm using that MQTT-Thing for many other stuff in my shack. So other applications can also access this data.

**Reply by magicbug** [reply]
Probably a bit complex for most but do you have written up how this all works?

**Comment by magicbug** [comment]
Huge thankyou @timkirby and @poll-busily should mean people who email me directly can get some useful information 👍🏻

**Comment by k7ilo** [comment]
Has anyone gotten adifpush to work with JTDX on linux?  I am interested in the automatic push to Cloudlog.  Anytime I run JTDX, I get Parse failure for STATUS_MESSAGE_TYPE and HEARTBEAT_MESSAGE_TYPE errors.  When I run WSJT-X, all works fine.

Thanks

**Reply by magicbug** [reply]
@M0LTE might be able to help with that

**Reply by M0LTE** [reply]
It needs work on my side. Different frame format. Please raise an issue on the adifpush repo.

**Reply by k7ilo** [reply]
Thanks both of you for your responses.

Just submitted Tom
Thanks for looking into this.

Harold - K7ILO

**Reply by M0LTE** [reply]
Now implemented in latest adifpush, could do with a tester...

**Comment by 112bzo** [comment]
Yes Gridtracker do this, i have it also via Gridtracker running.

**Reply by M0LTE** [reply]
Gridtracker does work great, if you want to run gridtracker.
